renderer/viewer/lib/entities.ts

```diff
@@ -350,19 +350,24 @@ export class Entities extends EventEmitter {
       playerObject.skin.map = skinTexture
       playerObject.skin.modelType = inferModelType(skinTexture.image)
 
-      const earsCanvas = document.createElement('canvas')
-      loadEarsToCanvasFromSkin(earsCanvas, image)
-      if (isCanvasBlank(earsCanvas)) {
-        playerObject.ears.map = null
-        playerObject.ears.visible = false
-      } else {
+      let renderEars = this.viewer.world.config.renderEars || username === 'deadmau5'
```

Comment:
username === 'deadmau5'?

Reply:
Yes, the ears are normally only enabled for Deadmau5 in the client (it checks the username) and I saw no reason not to support that "feature" (as some might use it on NPCs/as a disguise or you might even run into them for real lol)

Reply:
Oh ok, I didn't know it. Didn't know that even the client has the check, yes it totally makes sense thanks for letting me know